sycl/test-e2e/ESIMD/slm_block_load_store.cpp

```diff
@@ -10,16 +10,11 @@
 //
 // UNSUPPORTED: esimd_emulator
 //
-// GPU driver had an error in handling of SLM aligned block_loads/stores,
-// which has been fixed only in "1.3.26816", and in win/opencl version going
-// _after_ 101.4575.
+// Note: "lin" format below is used for Win L0 as well.
 // REQUIRES-INTEL-DRIVER: lin: 26816, win: 101.4576
 //
-// Even though the driver we use in CI is now new enough so this test doesn't
-// skip itself, it still fails. Temporary XFAIL-ing it while it is being
-// investigated.
-// FIXME: enable it back after investigation, see intel/llvm#11358
-// XFAIL: linux
+// https://github.com/intel/llvm/issues/11358
```

Comment:
LGTM, spoke with andrei fofline, we need UNSUPPORTED because conditional XFAIL doesn't work with REQUIRES-INTEL-DRIVER